docs/setup/prerequisites.md

```diff
@@ -4,12 +4,25 @@ sidebar_position: 1
 
 # Prerequisites
 
-1. Visual Studio 2022 or Visual Studio Code. Visual Studio Code is recommended. If you choose Visual Studio instead, make sure you get the community edition and use version 2022. **2019 will not work, only 2022.**
+1. Download Visual Studio 2022 Community Edition
+   
+   - [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) (RECOMMENDED)
 
-    - [Visual Studio Code](https://code.visualstudio.com/?wt.mc_id=vscom_downloads)
-    - [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
+If you are a beginner, you want to use Visual Studio 2022 NOT Visual Studio CODE for the entirety of the setup. Remember, if you decide for Visual Studio 2022, you only need to do the Visual Studio 2022 parts of the tutorial and you can skip the (NOT Visual Studio CODE sections.
 
-2. .NET 6.0 SDK If you don't know what Arm64, x64, and x86 mean then google it, but you're probably x64.
+**WARNING**: You need to use Visual Studio 2022 Community (higher editions such as professional also work, but are not required).
+    **Versions below 2022 will NOT work**
+
+You can skip to step 2 if you do not plan to use another environment than Visual Studio 2022.
+
+If you know what you are doing, we also have instructions how to setup the server using Visual Studio CODE but it is NOT recommended for beginners.
+You can download it here:
+
+- [Visual Studio Code](https://code.visualstudio.com/?wt.mc_id=vscom_downloads)
+
+In case you prefer a Visual Studio alternative, you can of course also use Rider. The largest difference between Rider and Visual Studio will be, perhaps, different namings for the menu options, but you should be able to follow along the Visual Studio Setup analogously.
```

Comment:
Rider is paid so not worth saying its a good alternative

Reply:
free for students tho, so I would leave it in, also other more experienced devs might be happy to hear they can use their favorite IDE without problems,